plugins/chat/assets/javascripts/discourse/services/chat-channels-manager.js

```diff
@@ -145,6 +155,16 @@ export default class ChatChannelsManager extends Service {
     );
   }
 
+  @cached
+  get directMessageChannelsWithActivity() {
```

Comment:
I wonder should this include channels with urgent badges too (ie. mentions or watched threads)?

Reply:
I assumed that urgent messages would be included in the unread count, but maybe it's possible they wouldn't be? Those are some complex queries that populate those values, so I can't say for certain! :D

Reply:
Yeah that's a fair point, these queries can probably be simplified/improved over time as they are pretty complex.